packages/popover/src/PopoverBase.tsx

```diff
@@ -1,44 +1,37 @@
-import React, { useState } from 'react';
+import * as React from 'react';
 import { usePopper } from 'react-popper';
-import PropTypes from 'prop-types';
 import { ClassManager, Constants } from '@axa-fr/react-toolkit-core';
-import PopoverPlacements from './PopoverPlacements';
+import { Placement } from '@popperjs/core';
 
-const propTypes = {
-  ...Constants.propTypes,
-  isOpen: PropTypes.bool.isRequired,
-  children: PropTypes.any,
-  placement: PropTypes.oneOf([
-    PopoverPlacements.top,
-    PopoverPlacements.bottom,
-    PopoverPlacements.left,
-    PopoverPlacements.right,
-  ]),
-};
 const defaultClassName = 'af-popover__container';
 const defaultProps = {
   ...Constants.defaultProps,
   className: defaultClassName,
-  placement: PopoverPlacements.top,
+  placement: 'top' as Placement,
```

Comment:
Same here, can you do Placement.top instead?

Reply:
I can't because it's a type, not an enum